Examples/Scripts/Python/full_chain_odd.py

```diff
@@ -157,25 +166,40 @@
         field,
         CKFPerformanceConfig(
             ptMin=1.0 * u.GeV if ttbar_pu200 else 0.0,
-            nMeasurementsMin=6,
+            nMeasurementsMin=7,
```

Comment:
I guess because of that @paulgessinger

Reply:
oh yeah sorry. but I guess it is the one below where it also changed from 6 to 7 right?

Reply:
here https://github.com/acts-project/acts/pull/1877/files#diff-218e044dd033aa9356a7aa4785f709eef7f2a72f1b7cb7e3db5a95006efc3b9bL170-L178

Reply:
but I guess this is also only for performance writing

Reply:
Yes, what is changing the physmon is this line : https://github.com/acts-project/acts/pull/1877/files#diff-311aa2ecef0d9d175b50ee63b8107c6ef3ef32fe6d3276d1ad462201f4b16891R127-R131